tests/test_device_properties.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for validating device properties for different basic data types."""
+import pytest
+
+from nidaqmx.constants import BusType, TriggerUsage
+
+
+class TestDevicePropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for different
+    data types of device properties.
+    """
```

Comment:
Remove the test class. Test functions should be sufficient.

Reply:
removed the class